webpack.config.js

```diff
@@ -45,22 +45,33 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader'),
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: 'css-loader'
+        }),
       },
       { test: /\.md/, loader: 'raw-loader' },
     ],
     // Silence warning https://github.com/localForage/localForage/issues/599
     noParse: [new RegExp('node_modules/localforage/dist/localforage.js')],
   },
   resolve: {
-    root: path.resolve(__dirname, 'app'),
-    extensions: ['', '.js', '.json'],
-    alias: { shared: path.resolve(__dirname, 'shared') }
+    modules: [
+      path.resolve(__dirname, 'app'),
+      'node_modules'
+    ],
+    alias: {
+      shared: path.resolve(__dirname, 'shared'),
+      'boundless-arrow-key-navigation': 'boundless-arrow-key-navigation/build',
```

Comment:
Any insight into why these modules need this now?

Reply:
I got just syntax errors, seems that it pulls the source by default and I excluded transpiling node modules. If this should not be the case, let me know

Reply:
no idea what was going on before, I doubt we were transpiling everything... or maybe I'm missing something

Reply:
Okay, looked into this - it seems that the `boundless` modules have a `module` key in the package json that points to uncompiled code and webpack3 is reading that. The correct way to fix for now is to add:

`mainFields: ["browser",  "main"]` within the resolve config alongside `alias`, you can then remove these items. I've tested this locally.

Reply:
thanks!